addons/intel/README.md

```diff
@@ -0,0 +1,109 @@
+# PTP Operator
+## Table of Contents
+
+- [E810 plugin](#ptp-operator)
+- [Quick Start](#quick-start)
+
+## E810 plugin
+Intel E810 plugin can be used to do hardware-specific configurations as required for e810 NICs, in order to use as a PTP grandmaster.
+
+
+
+## Quick Start
+
+#### Example config enable e810 plugin and doing configuration as GM.
+```
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: ptpconfig-gm
+  namespace: openshift-ptp
+spec:
+  profile:
+  - name: "profile1"
+    ...
+    ...
+    ......   
+    plugins:
+      e810:
+        enableDefaultConfig: false
```

Comment:
can we make this as by default  false? at this point this is usable only for non WPC or if you want to use ts2phc only , correct ?

Reply:
already by default is false